server/auth/store_test.go

```diff
@@ -114,12 +114,28 @@ func setupAuthStore(t *testing.T) (store *authStore, teardownfunc func(t *testin
 		t.Fatal(err)
 	}
 
+	// The UserAdd function cannot generate old etcd version user data (user's option is nil)
+	// add special users through the underlying interface
+	addUserWithNoOption(as)
```

Comment:
Nit: do we need to add a user with no `Options` for all test cases? I feel just adding it in `TestUserChangePassword` might be fine.

Reply:
I intend to add to the global test case, generate an old version of data, to avoid similar problems in the future, and at the same time all the current test cases are passed

Reply:
Understood, then it makes sense.